notebooks/Getting started - Exploring and Visualizing Data.ipynb

```diff
@@ -0,0 +1,336 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Loading, Visualizing, and Exploring the Data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This notebook will help you start \n",
+    "to explore the soil spectral data\n",
+    "through [pandas](https://pandas.pydata.org/pandas-docs/stable/), \n",
+    "[seaborn](http://seaborn.pydata.org/index.html), \n",
+    "and [matplotlib](https://matplotlib.org/index.html).\n",
+    "\n",
+    "All of these packages are well-documented and\n",
+    "easy-to-use, so they make for great tools\n",
+    "to use when first looking at data.\n",
+    "\n",
+    "A good resource for learning pandas \n",
+    "can be found [here](https://github.com/brandon-rhodes/pycon-pandas-tutorial)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "import matplotlib.pyplot as plt\n",
+    "import pandas as pd\n",
+    "import sklearn as skl\n",
+    "import numpy as np\n",
+    "\n",
+    "import seaborn as sns\n",
+    "sns.set(font_scale=2)\n",
+    "\n",
+    "%matplotlib inline"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Load the data and view the first few columns\n",
+    "\n",
+    "Pandas can read the contents of a csv file\n",
+    "directly into a pandas dataframe, just tell\n",
+    "it where to find the file."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data = pd.read_csv('../data/training.csv')\n",
```

Comment:
yea I'll change it... my original thought was that this first notebook would be super step-by-step, but don't think that will happen.  

Pretty sure anyone opening up our demo can figure out how to load a pandas dataframe without a special loader.